scripts/cmd/_run_enceladus.cmd

```diff
@@ -665,29 +659,20 @@ EXIT /B 0
 EXIT /B 0
 
 :temp_log_file
-    IF %DATE_PARSING_TYPE%==parser1 (
-        ::25.06.2021
-        SET yyyy=%date:~-4%
-        SET MM=%date:~3,2%
-        SET DD=%date:~0,2%
-    )
-    IF %DATE_PARSING_TYPE%==parser2 (
-        ::Fri 06/25/2021
-        SET YYYY=%date:~10,4%
-        SET MM=%date:~4,2%
-        SET DD=%date:~7,2%
-    )
-    IF %DATE_PARSING_TYPE%==parser3 (
-        :: 2021-06-25
-        SET yyyy=%date:~0,4%
-        SET MM=%date:~5,2%
-        SET DD=%date:~8,2%
+    ::Date&Time parsing
+    FOR /F "skip=1 tokens=1-6" %%A IN ('WMIC Path Win32_LocalTime Get Day^,Hour^,Minute^,Month^,Second^,Year /Format:table') DO (
+        if "%%B" NEQ "" (
```

Comment:
Again, naturally, I would have looked for cmd alternative for shell `myCommand | head -n2 | tail -n1`, but this seems to be simply available only in powershell 🙄

Reply:
I admit, I found this on the Internet, not my invention at all.